### ISSUE TITLE
Script does not work

### ISSUE THREAD

**Issue by CHJ85** [JDipi/Smugmug-Album-Downloader#1]
Hi there. Nothing happens when I click the "download album" button.
I looked at the code, and line 66 and 67 starts with "albumId" and "AlbumKey".
That might be the reason. Shouldn't those lines start with "let" or something?
Though I tried that, but still didn't work.

**Comment by JDipi** [comment]
Hi,

Can you please send me a link to the album you are trying to download? I have only used this on a specific sub-domain of the SmugMug site, so it will not work (yet) on other subdomains (see line 74 of the script). If you can send me the URL to the album, I can see about adding functionality to other subdomains. Or, you can try editing the API domain name on line 74 to match whatever domain you're using. See "Things I might add in the future" on the readme.

I also realized that I forgot to include one of the greasemonkey functions at the top of the script in this version, so I added that in.

**Comment by CHJ85** [comment]
Of course. manhattan.smugmug.com/

**Comment by JDipi** [comment]
@CHJ85 I re-wrote it so that it can access any Smugmug domain. Also, I saw that some of the galleries on that manhattan domain are password protected, and I am not sure if the script will be able to download them (assuming you have the password). Other than that, it seems to be working fine. 

I also added some basic error handling and logging to the UI.

Let me know if there are any more issues or any suggestions you have!
